lib/ladybug/energy_model/room.rb

```diff
@@ -77,9 +77,39 @@ def create_openstudio_object(openstudio_model)
           face = Face.new(face)
           openstudio_face = face.to_openstudio(openstudio_model)
           openstudio_face.setSpace(openstudio_space)
+
           nil
         end
 
+        openstudio_surfaces = openstudio_space.surfaces.each do |surface|
+          if surface.outsideBoundaryCondition == 'Adiabatic'
```

Comment:
I think you need an extra check here. You only want to do this in the event that the surface boundary condition is adiabatic AND there isn't already a construction hard-assigned to the surface. Otherwise, you might be overwriting a construction that is intended to be hard-assigned here from the honeybee schema.